### PR TITLE
[invite] feature/CAES-208: updated invite logic

### PR DESCRIPTION
**Pull request by ipopov4xxi**

### CODE REVIEW

File: src/Entity/Item.php [hunk 1]
```diff
@@ -276,4 +283,20 @@ public function removeTag(Tag $tag): void
...
+     */
+    public function getAccess(): ?string
+    {
+        return $this->access;
+    }
+
+    /**
+     * @param string $access
+     */
+    public function setAccess(string $access): void
```
Comment (aburov-4xxi) [inline src/Entity/Item.php]: ?string - because nullable=true

File: src/Form/Request/Invite/InviteCollectionRequestType.php [hunk 2]
```diff
@@ -24,6 +39,28 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'allow_add' => true,
                 'entry_type' => InviteType::class,
             ]);
+
+        $builder->addEventListener(FormEvents::SUBMIT, [$this, 'validateInvites']);
+    }
+
+    public function validateINvites(FormEvent $event)
```
Comment (aburov-4xxi) [inline src/Form/Request/Invite/InviteCollectionRequestType.php]: INvites -> Invites

File: src/Entity/Item.php (hunk 1 above)
Comment (ipopov4xxi) [reply src/Entity/Item.php]: Fixed

File: src/Form/Request/Invite/InviteCollectionRequestType.php (hunk 2 above)
Comment (ipopov4xxi) [reply src/Form/Request/Invite/InviteCollectionRequestType.php]: Fixed

**Pull request merged**